src/UIExplorer.elm

```diff
@@ -207,29 +208,36 @@ type alias Model a b c =
     }
 
 
+{-| Use this type to change the log whether using image url or a html element
```

Comment:
I think it's better to not expose constructors of this type and make it opaque. 
I like your idea of adding functions `UIExplorer.logoFromUrl` and `UIExplorer.logoFromHtml` that we can use to make the logo we want.

Reply:
small typo `Use this type to change the logo` (logo)